packages/core/src/utils/summarizer.code-context.md

```
@@ -0,0 +1,174 @@
+```typescript
+const compressed = await compressContextChunks(
+  chunks,        // Array of CodeContextChunk[]
+  targetTokens,  // Target token budget
+  geminiClient,  // GeminiClient instance
+  abortSignal,   // AbortSignal for cancellation
+  options        // Optional CodeContextOptions
+);
+```
+
+**Features:**
+- Sorts chunks by relevance score
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)